src/System.Reflection.Context/tests/CustomReflectionContextTests.cs

```diff
@@ -2,19 +2,44 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Linq;
 using Xunit;
 
 namespace System.Reflection.Context
 {
     public class CustomReflectionContextTests
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
-        public void InstantiateContext_Throws()
+        public void CustomContext()
+        {
+            var customReflectionContext = new MyCRC();
```

Comment:
This is a pretty complex set of code and I would highly recommend additional test coverage for it.

Reply:
right, I'll add some more. note that the public API surface is pretty small in comparison to the amount of code in the assembly.